src/map/skill.cpp

```diff
@@ -14207,6 +14200,10 @@ int skill_unit_onplace_timer(struct skill_unit *unit, struct block_list *bl, t_t
 					if (rnd()%100 < unit->val1)
 						skill_attack(BF_WEAPON,ss,&unit->bl,bl,sg->skill_id,sg->skill_lv,tick,0);
 					break;
+				case WL_COMET:
```

Comment:
Maybe check if NPC_COMET should be added here as well.
Maybe @Atemo or @mrjnumber1 know.